content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
-layers:
-  None
+custom:
+  webpack:
+    includeModules: true
 ```
 
-### What's next
+`serverless-webpack-prisma` automatically runs `prisma generate` so that you do not need to use the webpack script option separately as shown below.
 
-You can now test the deployment with a call to the status endpoint. Use the following command and replace `UNIQUE_IDENTIFIER` with the corresponding part of the URL from the output you got from `serverless deploy`.
-
-```terminal
-curl https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/
+```yaml file=serverless.yml
+custom:
+  webpack:
+    packagerOptions:
+      scripts:
+        - prisma generate
 ```
 
-This returns the following:
+`serverless-webpack-prisma` removes the unnecessary dependencies and packages the correct Prisma engine binary. Therefore, delete it in the `package.patterns` configuration in your `serverless.yml` file.
 
-```terminal no-copy
-{"up":true}
+```yaml file=serverless.yml
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
 ```
 
-## 6. Test your deployed REST API
-
-With the API base URL `https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/`, you can test the API's endpoints:
-
-| Endpoint      | Description                                                                                                              | Implementation            |
-| ------------- | ------------------------------------------------------------------------------------------------------------------------ | ------------------------- |
-| `GET /`       | Status                                                                                                                   | `handlers/status.js`      |
-| `GET /seed`   | Delete all database records and seed the database with test `users`, `profiles`, and `posts`. Returns the created users. | `handlers/seed.js`        |
-| `GET /users`  | Fetch all `users` in the database with their related `profiles`                                                          | `handlers/users.js`       |
-| `POST /users` | Create a single `users` in the database                                                                                  | `handlers/create-user.js` |
-| `GET /posts`  | Fetch all `posts` and their related `authors`                                                                            | `handlers/posts.js`       |
-
-To call the API, you can use curl. For example, the following command calls the `seed` endpoint. Replace `UNIQUE_IDENTIFIER` with the corresponding part of the URL from the output of `serverless deploy` in step 6.
+Now redeploy your application. To do so, run `serverless deploy`. The deployment output shows you that the correct Prisma engine binary is packaged and distributed, as follows:
```

Comment:
The output below does not actually show that the correct one is included, only that wrong ones are excluded. (Which might be something worth fixing in the plugin itself if this is indeed the case)